src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -207,7 +211,7 @@ public function normalize($object, string $format = null, array $context = [])
             $data = $this->updateData($data, $attribute, $this->serializer->normalize($attributeValue, $format, $childContext), $class, $format, $attributeContext);
         }
 
-        if (isset($context[self::PRESERVE_EMPTY_OBJECTS]) && !\count($data)) {
+        if ((isset($context[self::PRESERVE_EMPTY_OBJECTS])) && !\count($data)) {
```

Comment:
Is it on purpose or not?

Reply:
oups, this is an artefact of some test, I'll remove it :)